public-ui/src/app/views/ehr-view/ehr-view.component.ts

```diff
@@ -126,6 +127,10 @@ export class EhrViewComponent implements OnInit, OnDestroy {
   private searchCallback(results: any) {
     this.searchResult = results;
     this.items = this.searchResult.items;
+    this.standardConcepts = this.searchResult.standardConcepts;
```

Comment:
duplicate? Or does this return an empty array if there are none? We may not need the if's below.

Reply:
I added else part to set standard concept to empty array because i saw it getting the previously fetched standard concepts as part of next search.